### PR TITLE
Feature add:  switch for keeping lock screen shade

### PR DESCRIPTION
**Pull request by entr0pia**
ref: #361

### CODE REVIEW

**Comment by Mahmud0808** [comment]
What's with the strings? All are duplicates.

File: app/src/main/java/com/drdisagree/iconify/xposed/modules/QSTransparency.java [hunk 1]
```diff
@@ -64,9 +67,13 @@ protected void beforeHookedMethod(MethodHookParam param) throws Throwable {
                 String scrimState = getObjectField(param.thisObject, "mState").toString();
 
                 if (scrimState.equals("KEYGUARD")) {
-                    param.args[alphaIndex] = 0.0f;
+                    if(!keepLockScreenShade) {
+                        param.args[alphaIndex] = 0.0f;
+                    }
                 } else if (scrimState.contains("BOUNCER")) {
-                    param.args[alphaIndex] = (Float) param.args[alphaIndex] * keyguard_alpha;
+                    if(!keepLockScreenShade) {
```
Comment (Mahmud0808) [inline app/src/main/java/com/drdisagree/iconify/xposed/modules/QSTransparency.java]: I don't think you need to change this. This is the scrim shown in password/pattern screen.

**Comment by entr0pia** [comment]
> What's with the strings? All are duplicates.

I saw some untranslated words in `values-zh-rTW` and I thought it was a "style" of placeholder. They can be removed if not needed.

**Comment by Mahmud0808** [comment]
> > What's with the strings? All are duplicates.
> 
> I saw some untranslated words in `values-zh-rTW` and I thought it was a "style" of placeholder. They can be removed if not needed.

placeholders are taken care of automatically by crowdin.

File: app/src/main/java/com/drdisagree/iconify/xposed/modules/QSTransparency.java (hunk 1 above)
Comment (entr0pia) [reply app/src/main/java/com/drdisagree/iconify/xposed/modules/QSTransparency.java]: Tested again, it is indeed not necessary.

**Pull request merged**